sysprog21/semu
# Enable ISIG control mode of termios after exit

How to reproduce?
1. Run the emulation.
```
$ make check
```

2. Terminate the emulation via keyboard `CTRL + a + x`.

3. Run some shell interactive program (e.g., cat).
```
$ cat
```

4. Try to terminate the shell interactive program (e.g., cat) via keyboard `CTRL + c`.
The SIGINT cannot be caught and propagate to the `cat` process. Example output:
```
$ cat
apple
apple
^C^C^C^C^C
```

This patch fix this issue.
